### DISCUSSION TITLE
undefined symbol: absl::lts_20240116::log_internal::LogMessage& absl::lts_20240116::log_internal::LogMessage::operator<<<unsigned long, 0>(unsigned long const&) referenced by log_message.h:132 (/opt/vcpkg/installed/x64-linux/include/absl/log/internal/log_message.h:132) after switching from clang-17 to 18

### DISCUSSION THREAD

**Discussion by zlojvavan** [abseil/abseil-cpp#1743] [Q&A]
after upgrading ubuntu 23.10 to 24.04 and clang from 17 to 18 cannot build my grpc related projects with clang 18 (and newer) due to link time undefined symbol in debug config, please see [clang bugreport and workaround found](https://github.com/llvm/llvm-project/issues/102443)

wonder whether anyone successfully used clang-18 to build grpc/absl projects?

**Comment by mering** [comment]
See #1747 and llvm/llvm-project#102443
